cleanlab_studio/utils/data_enrichment/enrichment_utils.py

```diff
@@ -0,0 +1,68 @@
+import re
+from typing import List, Tuple, Union
+
+import pandas as pd
+
+from cleanlab_studio.errors import ValidationError
+
```

Comment:
can we add another standalone method to supplement `get_regex_match` that runs the regex on all rows of a DF column?

I can imagine somebody needs to iterate many times to get the regex right, they shouldn't be regenerating LLM outputs every time though